vsketch_cli/gui.py

```diff
@@ -7,15 +8,17 @@
 from .sketch_viewer import SketchViewer
 
 
-def show(path: str, second_screen: bool = False) -> int:
+def show(path: str, output_dir: Optional[str], second_screen: bool = False) -> int:
     if not QApplication.instance():
         app = QApplication()
     else:
         app = QApplication.instance()  # type: ignore
     app.setAttribute(Qt.ApplicationAttribute.AA_UseHighDpiPixmaps)
 
     # create widget
-    widget = SketchViewer(pathlib.Path(path))
+    widget = SketchViewer(
+        pathlib.Path(path), pathlib.Path(output_dir) if output_dir is not None else None
```

Comment:
If you propagate a `Path` instead of a `str`, you dont need this check.

BTW, if you want to convert `path` into a `Path` instead of a `str` for consistency, that would be great :)

Reply:
Ack, updated in latest revision.